README.md

```diff
@@ -1,12 +1,16 @@
-# PHP-dev
+# PHP-DEV
 
-## Links:
-
-Made to use with [pluswerk/docker-global](https://github.com/pluswerk/docker-global).
+The development of a website is simplified using PHP-DEV.
+It mainly differentiates whether an Apache or Nginx is used and which PHP version is used.
```

Comment:
The choice isn't made by the Dockerfile itself but rather by the user. This setup just gives the user the ability to choose between a different set of PHP versions and apache or nginx

Reply:
changed